App/View Controllers/Posts/GetOutFrogRefreshSpinnerView.swift

```diff
@@ -0,0 +1,116 @@
+//  GetOutFrogRefreshSpinnerView.swift
+//
+//  Copyright 2022 Awful Contributors. CC BY-NC-SA 3.0 US https://github.com/Awful/Awful.app
+
+import Lottie
+
+final class GetOutFrogRefreshSpinnerView: UIView, PostsPageRefreshControlContent {
+    private var animationView: LottieAnimationView?
+    
+    init(theme: Theme) {
+        animationView = .init(name: "frog")
```

Comment:
Lil swift thing: `animationView` can be a `let` and no need to make it optional. Then there's no need to unwrap it throughout :)

Reply:
Sorted, thanks!